libs/librrgraph/src/base/rr_graph_builder.h

```diff
@@ -264,6 +264,10 @@ class RRGraphBuilder {
         node_storage_.alloc_and_load_edges(rr_edges_to_create);
     }
 
+    inline void override_edge_switch(RREdgeId edge_id, RRSwitchId switch_id) {
```

Comment:
To maintain consistency with other functions, I think it would be better to add a Doxygen comment. (just add a brief description)